src/main/java/internetShop/dao/jdbc/ProductJdbcDao.java

```diff
@@ -0,0 +1,56 @@
+package internetShop.dao.jdbc;
+
+import internetShop.dao.ProductDao;
+import internetShop.entity.product.Product;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.support.JdbcDaoSupport;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+
+public class ProductJdbcDao extends JdbcDaoSupport implements ProductDao {
+
+    private RowMapper<Product> productRowMapper = (resultSet, i) -> {
+        Product product = new Product();
```

Comment:
add @Accesstors(chain = true) in Product()
and make this record like return new Product()
.setName()
.setPrice()
.setId();